### PR TITLE
Add Privacy Policy page

### PR DESCRIPTION
**Pull request by tylermercer**

### CODE REVIEW

File: src/Pages/EmailFlow/IntroDialog.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
```
Comment (bradenwatkins) [inline src/Pages/EmailFlow/IntroDialog.js]: This file isn't being used anymore right? I thought I deleted it.

**Comment by Bobleyl** [comment]
@tylermercer If you look on our Google Drive there is a privacy policy document.  Do you want to place that into the Privacy Policy page before pulling it in?

**Comment by bradenwatkins** [comment]
Sorry if this makes things more complicated, but it might make sense to stick our privacy policy on our landing site?

File: src/Pages/EmailFlow/IntroDialog.js (hunk 1 above)
Comment (tylermercer) [reply src/Pages/EmailFlow/IntroDialog.js]: Oh lemme check. I might have done some accidental file necromancy when I merged master in to this branch

**Comment by tylermercer** [comment]
> 
> 
> Sorry if this makes things more complicated, but it might make sense to stick our privacy policy on our landing site?

I actually had that same thought last week. I think it would make a lot more sense than what we have here

**Pull request closed without merging**